extensions/skyportal/static/js/components/SaveAlertButton.jsx

```diff
@@ -0,0 +1,238 @@
+import React, { useState, useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Checkbox from "@material-ui/core/Checkbox";
+import Button from "@material-ui/core/Button";
+import ButtonGroup from "@material-ui/core/ButtonGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+import ClickAwayListener from "@material-ui/core/ClickAwayListener";
+import Grow from "@material-ui/core/Grow";
+import Paper from "@material-ui/core/Paper";
+import Popper from "@material-ui/core/Popper";
+import MenuItem from "@material-ui/core/MenuItem";
+import MenuList from "@material-ui/core/MenuList";
+import { useForm, Controller } from "react-hook-form";
+
+import * as alertActions from "../ducks/alert";
+import * as sourcetActions from "../ducks/source";
+import FormValidationError from "./FormValidationError";
+
+const SaveAlertButton = ({ alert, userGroups }) => {
```

Comment:
What happens when a user tries to save an object with only PID=3 alerts to the sitewide group?

Reply:
Great point, addressed [here](https://github.com/fritz-marshal/fritz/pull/44/commits/947934a7fa14288b278daae1e88c5b1f9375dd39). It will now fail.